components/collection/voteDrop/DropContainer.vue

```diff
@@ -277,7 +286,7 @@ const mintedPercent = computed(() => {
 
 const hasUserMinted = computed(
   () =>
-    Boolean(collectionData.value?.nftEntitiesConnection?.totalCount) ||
+    collectionData.value?.nftEntitiesConnection?.edges?.[0]?.node?.id ||
```

Comment:
why?

Reply:
The nft id would be used here. I have renamed it
<img width="576" alt="image" src="https://github.com/kodadot/nft-gallery/assets/31397967/916a7dd0-018b-4f3e-ae5d-0490ae9f6389">